### PR TITLE
Fix exam selector option coloring

### LINKED ISSUE #23: "Take exam" subject dropdown isn't coloured properly
The issue mentioned by me in #21 was not fixed by #22.

I have some exams already subscribed to, but the subject dropdown shows all but one in white, and only one in yellow.
(The subjects which don't have any exam posted are properly removed from the list.)

### PR DESCRIPTION
**Pull request by solymosi**
Should fix #23 

@whisperity Can you pls test this?

### CODE REVIEW

File: npu.user.js [hunk 1]
```diff
@@ -1044,7 +1044,7 @@
           '<style type="text/css"> ' +
             '#h_exams_gridExamList_bodytable tr.gridrow_blue td ' +
             '{ ' +
-              'background: #F8EFB1 !important; ' +
+              'background-color: #F8EFB1 !important; ' +
```
Comment (solymosi) [inline npu.user.js:1047]: The `[+]` icon next to the rows is implemented as a cell background image :facepalm:
This change prevents the icon from being overwritten in subscribed exam rows.

File: npu.user.js [hunk 2]
```diff
@@ -1102,6 +1102,12 @@
           }
         });
 
+        /* From most to least important */
```
Comment (whisperity) [inline npu.user.js]: Perhaps you could add a little explanation **why** precedence is needed.

**Review by whisperity — APPROVED** [review]
Apart from the slight comment, it works!

File: npu.user.js (hunk 2 above)
Comment (solymosi) [reply npu.user.js]: Added a few comments that hopefully make it clear.

**Review by whisperity — APPROVED** [review]

**Comment by whisperity** [comment]
Let's roll. :wink:

**Comment by solymosi** [comment]
Released 🎉

File: npu.user.js (hunk 1 above)
Comment (whisperity) [reply npu.user.js:1047]: @solymosi Could you please implement this fix to the "my own exams" page too? It seems that the ➕ buttons are missing from that page too.

**Pull request merged**